ledger/committee/src/lib.rs

```diff
@@ -50,7 +50,7 @@ pub const MAX_DELEGATORS: u32 = 100_000u32;
 pub struct Committee<N: Network> {
     /// The committee ID, defined as the hash of the starting round, members, and total stake.
     id: Field<N>,
-    /// The starting round number for this committee.
+    /// The round number (of the block) at which this committee is created (always even).
```

Comment:
Should we also the comment for `Committee::starting_round()`? It's currently identical to the previous old comment (_"Returns the starting round number for this committee"_).

Reply:
Yes, pushed.